apps/balancer-tools/src/app/apr/api/(utils)/validate.ts

```diff
@@ -33,9 +33,34 @@ const isValidPoolType = (types: string[] | null) => {
 };
 
 const OptionalNullableString = z.string().nullable().optional();
-const OptionalNullableDate = OptionalNullableString.transform((dateStr) =>
-  dateStr ? new Date(dateStr + "T00:00:00.000Z") : null,
-);
+const OptionalNullableDate = OptionalNullableString.transform((dateStr) => {
```

Comment:
Changed so follow the [API Docs](https://www.notion.so/API-Documentation-f0732e25a1814b16aade0a41e3febc30) on the dates being "mm-dd-yyyy"

Reply:
ok to me, but notice how much harder this made the whole thing, and it's also not a standard format like the ISO 8601 options